src/Recipe.js

```diff
@@ -19,25 +19,54 @@ class Recipe {
       })
       return acc;
     }, 0) / 100;
-  };
+  }
 
   displayInstructions() {
-    return this.instructions.reduce((acc, instruction) => {
+    return JSON.stringify(this.instructions.reduce((acc, instruction) => {
       acc[instruction.number] = instruction.instruction
       return acc;
-    }, {})
-  };
+    }, {}))
+  }
 
-  displayRecipePage() {
-  return `
+  displayIngredients() {
+    return JSON.stringify(this.ingredients.reduce((acc, ingredient) => {
+      acc[ingredient.name] = ingredient.quantity.amount + ' ' + ingredient.quantity.unit; 
+      return acc;
+    }, {}))
+  }
 
+  displayRecipePage() {
+    return `
+    <section class='recipe'>
```

Comment:
Standard for HTML attributes should always be double quotes. You have a mix of single and double here, so I would fix all of the single quote attributes to use double quotes.

Reply:
Good call, thanks for catching that